layouts/partials/cover.html

```diff
@@ -0,0 +1,35 @@
+{{- $IsHome := .IsHome }}
+
+{{- with .cxt}} {{/* Apply proper context from dict */}}
+{{- if .Params.cover.image }}
+
+{{- $sizes := (slice "480" "720" "1080" "1500") }}
+
+{{- $src := cond (default true .Site.Params.responsiveImages) (.Page.Resources.GetMatch (printf "*%s*" (.Params.cover.image))) false }}
+{{- $fallback := cond (default false .Params.cover.relative) (path.Join .RelPermalink .Params.cover.image) (.Params.cover.image) -}}
+
+{{- $addLink := and (not $IsHome) (default true .Site.Params.linkFullImages) -}}
+{{- $responsiveImg := and $src (default false .Params.cover.relative) -}}
+
+<figure class="entry-cover">  
+{{- if $responsiveImg -}}
+    {{- if $addLink }}<a href="{{ $src.RelPermalink }}" target="_blank" title="Original image">{{ end }}
+    <img sizes="(min-width: 768px) 720px, 100vw"
+    srcset='{{- range $sizes -}}
+        {{ if ge $src.Width . }}{{ ($src.Resize (printf "%sx" .)).RelPermalink }} {{ (printf "%sw" .) }},{{ end }}
+    {{- end -}} {{$src.RelPermalink}} {{printf "%dw" ($src.Width)}}'
+    {{- if ge $src.Width "720" }}src="{{ ($src.Resize "720x").RelPermalink }}"
+    {{- else }}src="{{ $src.RelPermalink }}" {{ end }}{{ with .Params.cover.alt }} alt="{{ . | plainify }}"{{ end }}>
+    
+{{- else }}{{/* Fall back to full image */}}
+    {{- if $addLink }}<a href="{{ $fallback | absURL }}" target="_blank" title="Original image">{{ end }}
```

Comment:
Let the `title` be `caption` else `alt`.
"Original image" won't look good for multilingual content

Reply:
Done.